src/test/java/net/sf/jabref/importer/fileformat/MedlinePlainImporterTest.java

```diff
@@ -0,0 +1,122 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+/**
+ * Test class for the MedlinePlain importer.
+ */
+
+public class MedlinePlainImporterTest {
+
+    private MedlinePlainImporter importer;
+
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        importer = new MedlinePlainImporter();
+    }
+
+    @Test
+    public void testIsRecognizedFormat() throws Exception {
+
+        List<String> list = Arrays.asList("CopacImporterTest1.txt", "CopacImporterTest2.txt", "IsiImporterTest1.isi",
+                "IsiImporterTestInspec.isi", "IsiImporterTestWOS.isi", "IsiImporterTestMedline.isi",
+                "MedlinePlainImporterTestEmpty.txt");
+
+        //Test with txt Files which are not in recognized Format
+        for (String str : list) {
+            try (InputStream is = MedlinePlainImporter.class.getResourceAsStream(str)) {
+                Assert.assertFalse(importer.isRecognizedFormat(is));
+            }
+        }
+
+        list = Arrays.asList("MedlinePlainImporterTest.txt", "MedlinePlainImporterTest1.txt",
+                "MedlinePlainImporterTest2.txt", "MedlinePlainImporterTest3.txt", "MedlinePlainImporterTest4.txt",
+                "MedlinePlainImporterTest5.txt");
+
+        //Test with txt File which is in recognized Format
+        for (String str : list) {
+            try (InputStream is = MedlinePlainImporter.class.getResourceAsStream(str)) {
+                Assert.assertTrue(importer.isRecognizedFormat(is));
+            }
+        }
+    }
+
+    @Test
+    public void testImportMultipleEntriesInSingleFile() throws Exception {
+        try (InputStream is = MedlinePlainImporter.class.getResourceAsStream("MedlinePlainImporterTest.txt")) {
+            //create entries list from the txt file
+            List<BibEntry> entries = importer.importEntries(is, new OutputPrinterToNull());
+            Assert.assertEquals(7, entries.size());

```

Comment:
If there are 7 entries then you should also test that all entries are parsed correctly and not just the first two. Probably it is more convenient to reduce the number of entries in the test file and thus simplify the test case.